vendor/engines/secure_rooms/db/migrate/20170301120659_create_card_reader_and_control_device.rb

```diff
@@ -0,0 +1,19 @@
+class CreateCardReaderAndControlDevice < ActiveRecord::Migration
+
+  def change
+    create_table :secure_rooms_control_devices do |t|
+      t.references :product, null: false
+      t.foreign_key :products
```

Comment:
So funny story: Oracle doesn't automatically create an index on this (or at least it didn't the last time I tried). We can leave it out for now to see what happens when we pull it down into NU, but I wanted to call that out.